nuntium/models.py

```diff
@@ -151,6 +152,24 @@ class WriteItInstanceConfig(models.Model):
         help_text=_("Messages pushed to the api should \
             be confirmed automatically"), default=True)
 
+    custom_from_domain = models.CharField(max_length=512, null=True, blank=True, default=None)
```

Comment:
I wonder if the `default=None`s are really necessary here? Is that not the default default?